### PR TITLE
Fix/minor fixes

### PR DESCRIPTION
**Pull request by galimba**
* Renames `Pool` to `Tanda`
* Reworks color scheme.

image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,5 +40,12 @@ To run the local server, run:
 $ yarn dev
 ```
 
+To run locally but connected to the testnet:
+
+```
+$ yarn build
```
Comment (pmprete) [inline README.md]: yarn build is only necesary to run before deploying a new version, yarn start is enough for local development

**Review by pmprete — DISMISSED** [review]
Use the Cointanda BetaTest logo.
Other than that and the comment on how to run locally, it looks good

File: README.md (hunk 1 above)
Comment (galimba) [reply README.md]: soled

**Review by pmprete — APPROVED** [review]
LGTM

**Pull request merged**